systest/Makefile

```diff
@@ -8,6 +8,8 @@ test_pod_name ?= systest-$(version_info)-$(shell date +'%s')
 keep ?= false
 clusters ?= 1
 size ?= 10
+psize ?= 1
+retries ?= 0
```

Comment:
why it make sense to set retries in env variables? currently env variables control only environment and resources, they weren't meant to parametrize test logic

would it work to provide some reasonable number of default retries and remove this variable completely?

Reply:
hehe... i really have no idea what the intended design for env variables was in the systest. 
now i know. done.